src/main/java/core/basesyntax/service/FileService.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.dao.FruitTransactionDao;
+import core.basesyntax.dao.FruitTransactionDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public class FileService {
+    private static final String INPUT_FILE_NAME = "src/input.csv";
+    private static final String OUTPUT_FILE_NAME = "src/output.csv";
+    private FruitDao fruitDao;
+    private FruitTransactionDao fruitTransactionDao;
+
+    public FileService() {
+        this.fruitDao = new FruitDaoImpl();
+        this.fruitTransactionDao = new FruitTransactionDaoImpl();
+    }
+
+    public List<FruitTransaction> readFromFile() {
```

Comment:
make a proper name of the file or return plain text and then convert it into a transactions list. Your code is hard to read

Reply:
not fixed

Reply:
not fixed, please review the comments carefully, some of them get collapsed and hidden if there are a lot of them

Reply:
I di

> not fixed, please review the comments carefully, some of them get collapsed and hidden if there are a lot of them

I don't quite understand what should I do. Should I return string in readFromFile or readFromFile must read only one line or readFromFile must read the file only once?